cf_xarray/accessor.py

```diff
@@ -1741,6 +1742,12 @@ def guess_coord_axis(self, verbose: bool = False) -> DataArray | Dataset:
                 continue  # prevent second detection
 
             for name, pattern in regex.items():
+                var_is_axis = var in sum(obj.cf.axes.values(), [])
+                var_is_coord = var in sum(obj.cf.coordinates.values(), [])
```

Comment:
```suggestion
```

Isn't this the same as above?

Reply:
yes, but we need to re-assess whether it's a coord in the for loop, otherwise multiple guesses are allowed.
I was undecided whether to copy and paste, create a function just for this, or maybe add a couple of private methods:
`obj.cf._var_is_coord(var)`

Reply:
Let's just add a comment. It's a single line that's pretty readable so no need for a function

Reply:
done